Pod/Classes/Pipes.swift

```diff
@@ -52,9 +52,8 @@ Try piping the optional input into the function on the right.
 
 Result is nil if lhs is nil. Otherwise, same as `|>`
 */
-public func |?> <A, B>(lhs: A?, rhs: A -> B) -> B? {
-    return lhs
-        |> map { rhs($0) }
+public func |?> <A, B>(lhs: A?, @noescape rhs: A -> B) -> B? {
+  return lhs.map(rhs)
```

Comment:
Please double check this change against previous Swift versions.
If I remember correctly, it used to be that you had to wrap `lhs` into `Optional(  )`, which was the reason for implementing it by using the `|>` operator.